src/doughnut/signingMethods/1/index.js

```diff
@@ -6,8 +6,12 @@ const {
   naclVerify,
 } = require("@polkadot/util-crypto");
 
+async function sign(payload, keypair) {
```

Comment:
this change doesn't make any real differences

Reply:
it's for keep the interface same with schnorrkel. `async` makes it returns Promise